### PR TITLE
Update intro text.

### PR DESCRIPTION
**Pull request by lliss**
Replace dummy text with real intro text.

Connects #137 

We already had a place for this, so I replaced the dummy text with this intro.

screen shot 2015-08-18 at 12 50 50 pm

screen shot 2015-08-18 at 12 52 00 pm

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -40,8 +40,11 @@
             <div class="modaloverlay-container">
                 <div id="introduction-modaloverlay" class="modaloverlay">
                     <div class="modaloverlay-content">
-                        <h1>A look through the past</h1>
-                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsum eligendi harum beatae autem velit corporis, dolor consequuntur est, saepe sit dolorem nemo animi sequi quas non inventore pariatur. Quasi, magnam!</p>
+                        <p>Welcome to the Fairmount Water Works' "Urban Water Revealed Prototype app." This work in progress is meant to give Philadelphians and visitors to our city self-guided access to the grounds of the historic Water Works complex through space and time.</p>
+                        <p>Please traverse our landscape to discover viewing points as portals that will take you through time to better understand the 19th century landmark innovation before you and its continued significance in the global water story.</p>
+                        <p>Enjoy!</p>
+                        <p>Karen Young &mdash; Executive Director, The Fairmount Water Works</p>
+</p>
```
Comment (caseypt) [inline src/index.html]: Indentation.

Comment (caseypt) [inline src/index.html]: Or a stray `<p>` tag.

**Comment by caseypt** [comment]
+1

File: src/index.html (hunk 1 above)
Comment (lliss) [inline src/index.html]: Whoops. Rebased.

**Pull request merged**